src/sys.c

```diff
@@ -231,6 +232,231 @@ JL_DLLEXPORT double jl_stat_ctime(char *statbuf)
     return (double)s->st_ctim.tv_sec + (double)s->st_ctim.tv_nsec * 1e-9;
 }
 
+JL_DLLEXPORT int jl_os_get_passwd(uv_passwd_t *pwd, size_t uid)
+{
+#ifdef _OS_WINDOWS_
+  return UV_ENOTSUP;
+#else
+  // taken directly from libuv
```

Comment:
Can we call libuv instead of copying the code?

Reply:
I added an extra argument, so we need to upstream this first.